demo/src/demos/svg.rs

```diff
@@ -887,7 +912,8 @@ impl App for Svg {
                     ..Default::default()
                 }),
             });
-
+            layer.set_transform(transform);
```

Comment:
You can probably add this on line 907:

```rust
layer.insert(path).set_transform(transform).set_props(...
```

Reply:
Yeah. I had my translation matrix wrong originally (I filled it out in a row major way, so had moved it out for debugging).

I think both are equivalent, but I've moved it to there.